### PR TITLE
Add ogp settings

### PR DESCRIPTION
**Pull request by rnitame**

### CODE REVIEW

File: layouts/partials/header.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html>
-    <head>
+    <head prefix="og: http://ogp.me/ns# fb: http://ogp.me/ns/fb# article: http://ogp.me/ns/article#">
```
Comment (rnitame) [inline layouts/partials/header.html]: ref: https://developers.facebook.com/docs/reference/opengraph/object-type/article/

Comment (jbub) [inline layouts/partials/header.html]: 1. can you please provide more context why is this needed (cant tell from the link) ? 
2. we are including open graph stuff only when `opengraph = true` param is enabled
 (SEE https://github.com/jbub/ghostwriter/blob/master/layouts/partials/header.html#L13)

thanks for your contribution!

Comment (rnitame) [inline layouts/partials/header.html]: > can you please provide more context why is this needed (cant tell from the link) ?

This error appeared in Facebook debugger when prefix not set.

image

> we are including open graph stuff only when opengraph = true param is enabled (SEE 

oh...sorry. I missed it.
How to set OGP was difficult for me to understand.
It becomes easy to understand if you add it to default.md

Comment (jbub) [inline layouts/partials/header.html]: 1. Well that error is shown when you do not provide the image for a post. If you set image for your post/page, we will display it:
https://github.com/jbub/ghostwriter/blob/master/layouts/partials/opengraph.html#L5-L9
2. I have just read about the prefix attribute and it looks like we do not need it (yet) as per: 
https://www.w3.org/TR/xhtml-rdfa-primer/#h-note8

Comment (jbub) [inline layouts/partials/header.html]: So what we need to do is only include the prefix when `opengraph` param is enabled, i think it is a good idea to include it, but since the `og` prefix is part of the initial context for RDF, it will work without it.

Comment (jbub) [inline layouts/partials/header.html]: Somethink like:

`<html{{ if .Site.Params.opengraph }} prefix="og: http://ogp.me/ns#"{{ end }}>`

should do, feel free to update the PR, or if it is too hard, i can do that :)

Comment (rnitame) [inline layouts/partials/header.html]: Thank you very much in detail !!
I've updated the PR, please check 🙇

File: archetypes/default.md [hunk 2]
```diff
@@ -3,4 +3,5 @@ title = ""
 description = ""
 author = ""
 tags = []
+images = [""]
```
Comment (jbub) [inline archetypes/default.md]: This one is not needed.

File: layouts/partials/header.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html>
-    <head>
+    <head {{ if .Site.Params.opengraph }} prefix="og: http://ogp.me/ns#" {{ end }}>
```
Comment (jbub) [inline layouts/partials/header.html]: Please move this to `<html>` tag as per http://ogp.me/, also the spaces inside the if are redundant:

`<html{{ if .Site.Params.opengraph }} prefix="og: http://ogp.me/ns#"{{ end }}>`

**Comment by rnitame** [comment]
@jbub I've modified, please check

**Comment by jbub** [comment]
Thank you!

**Comment by rnitame** [comment]
@jbub Thanks!! Happy new yearrrrrrr!!
